data/scripts.js

```diff
@@ -488,7 +488,7 @@ let BattleScripts = {
 			let i;
 			for (i = 0; i < hits && target.hp && pokemon.hp; i++) {
 				if (pokemon.status === 'slp' && !isSleepUsable) break;
-
+				if (move.parentalBondModifier && i > 0) move.basePower = move.basePower * move.parentalBondModifier;
```

Comment:
I'm not sure this would put the modifier in the right place relative to the other modifiers. @Marty-D?

Reply:
Parental Bond appears to happen between the multi-target modifier and the weather modifier. I'll edit in the research link in a minute.